virttest/libvirt_xml/devices/interface.py

```diff
@@ -110,11 +111,39 @@ def __init__(self, type_name, virsh_instance=base.base.virsh):
                                tag_name='virtualport', attribute='type')
         accessors.XMLElementDict('alias', self, parent_xpath='/',
                                  tag_name='alias')
+        accessors.XMLElementList(property_name='ips',
+                                 libvirtxml=self,
+                                 forbidden=None,
+                                 parent_xpath='/',
+                                 marshal_from=self.marshal_from_ip,
+                                 marshal_to=self.marshal_to_ip)
+
     # For convenience
     Address = librarian.get('address')
 
     Filterref = librarian.get('filterref')
 
+    @staticmethod
+    def marshal_from_ip(item, index, libvirtxml):
```

Comment:
change marshal_from_ip to marshal_from_ips since attribute name is ips

Reply:
updated